### ISSUE TITLE
Sometimes no Wifi or/and LAN after suspend by NV41 from novacustom

### ISSUE THREAD

**Issue by hubbsterx** [QubesOS/qubes-issues#9333]
### Qubes OS release
R4.2.1

### Brief summary

I bought the novacustom NV41 laptop with coreboot and heads (0.9.1) and installed QubesOS on it (without kernel-latest option). The main Template is fedora39-xfce (I will update it next time to the 40). Specially NV41 was reported with working suspend mode.

The network devices are:
Intel Alder Lake-P PCH CNVi WiFi (so, the AX211)
Realtek Semiconductor RTL 8111/8168/8411

As I saw in the news from QubesOS in 2023, AX211 was not under officially supported wifi devices.

I tried the following points with different kernels (6.6.31 / 29 and 21). I even installed the kernel-latest (6.9.2) to check it, because I red in the news thread from Qubes, that it should be installed with this option. Later somebody in the Qubes forum said, that there is no need for it anymore (it would be good to update it in some info about NV41). So now, I use the standard 6.6.31 kernel.

### Steps to reproduce

I'm in the working system with sys-net and some other VMs opened, then I close my laptop and after some time open it again. Then I get the problem describes on actual behavior.
If I restart the sys-net, everything works again.

### Expected behavior

Everything should work as before suspend.

### Actual behavior

Sometimes (I could not reproduce it 100% or even 70%) there is no Wifi and/or LAN if I click on the icon above. I can see the networks if I want to edit them, but they can not be chosen to connect, for they are just not there. Sometimes the networks are there as expected. So it doesn't seem to be a fully reproducible problem.

**Comment by hubbsterx** [comment]
I added the issue also on dasharos channel: https://github.com/Dasharo/dasharo-issues/issues/921

**Comment by marmarek** [comment]
Can you post kernel logs from sys-net when it happens? Just save `sudo dmesg` output to a file.
If there is anything about "page allocation failure", you can try increasing sys-net's memory in its settings.

**Comment by marmarek** [comment]
This might be related to https://github.com/QubesOS/qubes-issues/issues/9335 (but no diagnosis there either).

**Comment by hubbsterx** [comment]
> Can you post kernel logs from sys-net when it happens? Just save `sudo dmesg` output to a file. If there is anything about "page allocation failure", you can try increasing sys-net's memory in its settings.

after some time without problems (I had the laptop on external monitor and clicked always suspend mode before closing the laptop, maybe it makes some difference?) I got the issue again (as I just used it on it's own. So just after some times closing the laptop (without clicking suspend before it).

Here is dmesg from sys-net after crashing the network:
[dmesg.txt](https://github.com/user-attachments/files/16134662/dmesg.txt)
As it seems there are some page allocation failures:
`[ 5708.506634] kworker/1:3: page allocation failure: order:5, mode:0xcc0(GFP_KERNEL), nodemask=(null),cpuset=/,mems_allowed=0`

But also much more failures/errors.

So maybe it will bring some light to this issue.
